book/src/sprites/orthographic_camera.md

```diff
@@ -0,0 +1,72 @@
+# Orthographic Camera
+
+Finally, you need to tell Amethyst to draw in 2D space. This is done by creating an entity with a `Camera` component using orthographic projection. For more information about orthographic projection, refer to the [OpenGL documentation][opengl_ortho].
+
+The following snippet demonstrates how to set up a `Camera` that sees entities within screen bounds, without culling entities based on Z position:
+
+```rust,no_run,noplaypen
+# extern crate amethyst;
+use amethyst::core::cgmath::{Matrix4, Ortho, Vector3};
+use amethyst::core::transform::GlobalTransform;
+# use amethyst::prelude::*;
+use amethyst::renderer::{
+    Camera, Projection, ScreenDimensions
+};
+
+#[derive(Debug)]
+struct ExampleState;
+
+impl<'a, 'b> SimpleState<'a, 'b> for ExampleState {
+    fn on_start(&mut self, mut data: StateData<GameData>) {
+        // ...
+
+        self.initialize_camera(&mut data.world);
+    }
+}
+
+impl ExampleState {
+    fn initialize_camera(&mut self, world: &mut World) {
+        let (width, height) = {
+            let dim = world.read_resource::<ScreenDimensions>();
+            (dim.width(), dim.height())
+        };
+
+        // Camera translation from origin.
+        //
+        // The Z coordinate of the camera is how far along it should be before it faces
+        // the entities. If an entity's Z coordinate is greater than the camera's Z
+        // coordinate, it will be culled.
+        //
+        // By using `::std::f32::MAX` here, we ensure that all entities will be in the
+        // camera's view.
+        let translation = Matrix4::from_translation(Vector3::new(
+            0.0,
+            0.0,
+            ::std::f32::MAX,
+        ));
+        let global_transform = GlobalTransform(translation);
+
+        let camera = world
+            .create_entity()
+            .with(Camera::from(Projection::Orthographic(Ortho {
+                left: 0.0,
+                right: width,
+                top: height,
+                bottom: 0.0,
+                near: 0.0,
+                // The distance that the camera can see. Since the camera is moved to
+                // the maximum Z position, we also need to give it maximum Z viewing
+                // distance to ensure it can see all entities in front of it.
+                far: ::std::f32::MAX,
```

Comment:
Logically, this won't work if entities are in a negative z position, as MAX-MAX = 0 and not negative MAX.

Reply:
hm, should I go with `f32::INFINITY` for both? still have to test it, but theoretically that would be unlimited both ways

Reply:
maybe
it would be worth testing to see what happens

Reply:
These values wont work at all in practice, we are still using floats here. Just keep the camera at z=0 and set near/far to -10/10 or sth like that.

Reply:
The -10 wouldn't work unless this is removed from [`sprite_visibility.rs`](https://github.com/amethyst/amethyst/blob/1e36735c30eea2944bfbc0ce1b94b1a0ed14e5ea/amethyst_renderer/src/sprite_visibility.rs#L89-L90) right:

```rust
// filter entities behind the camera
.filter(|c| c.from_camera.dot(camera_backward) < 0.)
```

because that takes camera position into account and culls entities

Reply:
Spent a few hours on this (removed the z culling) and couldn't solve it. See [this monologue](https://discordapp.com/channels/425678876929163284/425679992244928512/500392571135066127) when I was describing the issue.

I'll update the float values to be more sensible, but shall leave it to someone else more experienced to solve negative z coordinate rendering when the camera isn't transformed.